src/platform/screen.h

```diff
@@ -18,7 +18,8 @@ void platform_screen_center_window(void);
 void platform_screen_recreate_texture(void);
 #endif
 
-void platform_screen_render(void);
+void platform_screen_render(int update_screen);
```

Comment:
Can we please eliminate this boolean flag by creating two functions?
Perhaps `platform_screen_draw_and_render(void)` (original code) and `platform_screen_render(void)` with just the `SDL_RenderPresent()`.

For rationale, see Clean Code by Robert C. Martin, or [this article](https://martinfowler.com/bliki/FlagArgument.html) by Martin Fowler.

Reply:
I completely agree. I was thinking about doing just that when I had some time 😉 

I was thinking about `platform_screen_update` and `platform_screen_render`. Normal code would call both functions, the vita keyboard code would just need to call `render`.

Reply:
Sounds good 👍